apps/accounts/admin.py

```diff
@@ -87,4 +89,17 @@ class UserIdentificationLabelAdmin(admin.ModelAdmin):
     ordering = ("weight", )
 
 
-admin.site.register(UserIdentificationLabel, UserIdentificationLabelAdmin)
+@admin.register(UserPasswordDescriptor)
+class UserPasswordDescriptorAdmin(admin.ModelAdmin):
+
+    list_display = ('user', 'date', 'salt', 'iterations')
+    list_filter = ('date', )
+    ordering = ('date', )
+
+
+@admin.register(PastPassword)
+class PastPasswordAdmin(admin.ModelAdmin):
+
+    list_display = ('userpassword_desc', 'password', 'date_created', )
+    list_filter = ('date_created', )
+    ordering = ('date_created', )
```

Comment:
@JFU-GIT @dtisza1 Is this worth exposing in the admin?

Reply:
@njdister @JFU-GIT Good question!  I know this was helpful for development, but I'm not sure if it would be useful in the ADMIN?

Reply:
this was 'copied' from what the 3rd party package does, if there is no much value, we could remove it to keep ADMIN UI concise.